developer-extension/src/panel/hooks/useEvents/eventFilters.ts

```diff
@@ -75,18 +75,42 @@ function filterOutdatedVersions(events: SdkEvent[]): SdkEvent[] {
 
 function parseQuery(query: string) {
   const queryParts = query
-    .split(' ')
+    .split(/(?<!\\)\s/gm) // Hack it to escape whitespace with backslashes
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏ nice!